demo.sh

```diff
@@ -1,2 +1,7 @@
 this is demo.updated data here
-i am adding this for Pull Request
+i am adding this for Pull Request from UAT to master
+
+i have added this is extra line
+please do not merge this 
+line in 
+the master branch
```

Comment:
remove empty line at 3
remove lines from 4 to 7
```suggestion

i have added this is extra line
please do not merge this 
line in 
the master branch
```